tools/slintpad/src/lsp.ts

```diff
@@ -213,8 +213,15 @@ class PreviewerBackend {
                     });
                 },
             );
-            this.#instance.set_design_mode(this.#picker_mode);
+            instance.set_design_mode(this.#picker_mode);
+        })
+    }
+
+    private async with_instance<R>(callback: (instance: slint_preview.WrappedInstance) => R): Promise<R | null> {
```

Comment:
the warning looks indeed like something's fishy.

Reply:
The warning is bogus AFAICS. Watch how eslint complains in this commit:

https://github.com/slint-ui/slint/pull/2879/commits/2681768d10e45d04987e58b116020a3f5eee82e6

claiming that `instance` is unused here:

```typescript
type InstanceCallback<R> = (_instance: slint_preview.WrappedInstance) => R;
```

and https://github.com/slint-ui/slint/pull/2879/commits/309ff0f3aec973f474c4eada3d13558168edc87a will silence it again.